edb/lib/schema.edgeql

```diff
@@ -129,6 +129,7 @@ CREATE TYPE schema::TupleElement EXTENDING std::BaseObject {
 
 
 CREATE TYPE schema::Tuple EXTENDING schema::CollectionType {
+    CREATE REQUIRED PROPERTY named -> bool;
```

Comment:
If we have a test for introspecting tuples please enhance it with this prop check

Reply:
We don't specifically have a test that introspects Tuples

(In general there are a lot of specific things in `schema` that don't have tests directly for them. I think that all the mechanisms get tested, but not each individual field. Stuff like exposing a bool field is really just a config change on pretty well tested code paths.)

Reply:
fair enough